app/views/movies/_movie_partial_loop.html.erb

```diff
@@ -1,24 +1,20 @@
 <div class="row tile-container">
   <% movies.each do |movie| %>
     <article class="tile-cover-pic">
-      <!-- MODAL TRIGGER -->
+      <!-- MODAL TRIGGER 1 of 2 -->
+      <!-- On Desktop: Always display the modal -->
       <p id="modal-trigger">
-        <!-- Always display the modal on desktop -->
         <%= link_to image_for(movie), movie_modal_path(tmdb_id: movie.tmdb_id, list_id: @list&.id),
                                       remote: true,
                                       data: { target: "#myModal_#{movie.tmdb_id}" },
                                       id: "modal_link_#{movie.tmdb_id}",
                                       class: 'modal-linker'
         %>
       </p>
-
+      <!-- MODAL TRIGGER 2 of 2 -->
+      <!-- On Mobile: skip the modal and go straight to a show page -->
       <p id="modal-trigger-mobile">
-        <!-- Get to a show page on mobile regardless of movie object persistence -->
-        <% if movie.is_a?(Movie) %>
-          <%= link_to image_for(movie), movie_path(movie) %>
-        <% else %>
-          <%= link_to image_for(movie), movie_more_path(tmdb_id: movie.tmdb_id) %>
```

Comment:
All `movie`s are instances of `MovieSearchResult`, so we don't need to check. The controller action for the `movie_more_path` is already handling the question of "should i display a persisted movie or hit the api to get fresh data?" with its use of `GuaranteedMovie`.